### PR TITLE
tcpreplay 4.3.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
<details>
<summary>license header ref</summary>

```
/*
 *   Copyright (c) 2001-2010 Aaron Turner <aturner at synfin dot net>
 *   Copyright (c) 2013-2018 Fred Klassen <tcpreplay at appneta dot com> - AppNeta
 *
 *   The Tcpreplay Suite of tools is free software: you can redistribute it
 *   and/or modify it under the terms of the GNU General Public License as
 *   published by the Free Software Foundation, either version 3 of the
 *   License, or with the authors permission any later version.
 *
 *   The Tcpreplay Suite is distributed in the hope that it will be useful,
 *   but WITHOUT ANY WARRANTY; without even the implied warranty of
 *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *   GNU General Public License for more details.
 *
 *   You should have received a copy of the GNU General Public License
 *   along with the Tcpreplay Suite.  If not, see <http://www.gnu.org/licenses/>.
 */
```

</details>

File: Formula/tcpreplay.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Tcpreplay < Formula
   desc "Replay saved tcpdump files at arbitrary speeds"
   homepage "https://tcpreplay.appneta.com/"
-  url "https://github.com/appneta/tcpreplay/releases/download/v4.3.3/tcpreplay-4.3.3.tar.gz"
-  sha256 "ed2402caa9434ff5c74b2e7b31178c73e7c7c5c4ea1e1d0e2e39a7dc46958fde"
-  license "GPL-3.0"
+  url "https://github.com/appneta/tcpreplay/releases/download/v4.3.4/tcpreplay-4.3.4.tar.gz"
+  sha256 "ee065310806c22e2fd36f014e1ebb331b98a7ec4db958e91c3d9cbda0640d92c"
+  license "GPL-3.0-or-later"
```
Comment (carlocab) [inline Formula/tcpreplay.rb:6]: It's got a few more licenses: https://metadata.ftp-master.debian.org/changelogs//main/t/tcpreplay/tcpreplay_4.3.3-2_copyright

`BSD-4-Clause`, `ISC`, `BSD-2-Clause`, `BSD-3-Clause`

**Comment by chenrui333** [comment]
```
configure: error: in `/private/tmp/tcpreplay-20210612-22759-zqdznu/tcpreplay-4.3.4':
configure: error: C compiler cannot create executables
```

File: Formula/tcpreplay.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/tcpreplay.rb]: included all the licenses.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**